panels/models.py

```diff
@@ -20,7 +25,7 @@ class Event(MagModel):
     description = Column(UnicodeText)
 
     assigned_panelists = relationship('AssignedPanelist', backref='event')
-    application = relationship('PanelApplication', backref='event')
+    applications = relationship('PanelApplication', backref='event')
```

Comment:
Since there can be multiple applications related to an event on the schedule, I changed this attribute to be pluralized.